src/components/FeeSummary/FeeSummary.vue

```diff
@@ -83,6 +93,10 @@ export default class FeeSummary extends Vue {
   @Prop({ default: '' })
   readonly errorMessage: string
 
+  /** Prop to indicate review mode. */
+  @Prop({ default: false })
+  readonly isSummaryMode: boolean
```

Comment:
Why not call it `isReviewMode` then?

Reply:
Will update comment, changed it summary after the fact to match the precedence set in the UI's

Reply:
Updated!